build.sbt

```diff
@@ -637,6 +646,19 @@ lazy val hasLargeIntegrationTests = Seq(
   javaOptions in (Test, run) += "-Xss4m"
 )
 
+lazy val nativeSettings = Seq(
+  scalaVersion := LatestScala211,
+  crossScalaVersions := List(LatestScala211),
+  scalacOptions -= "-Xfatal-warnings",
+  nativeGC := "immix",
+  nativeMode := "release",
+  nativeLinkStubs := false,
+  // These builds are published from my private fork of Scala Native
+  // https://github.com/xeno-by/scalapb/commits/topic/scalameta
```

Comment:
you should send a PR to scalapb.

Reply:
It's best to way until the next stable native release where there won't be a need to rebuild dependencies unless the IR changes.

Reply:
Sounds good